cliquet/statsd.py

```diff
@@ -33,6 +33,10 @@ def count(self, key, unique=None):
         else:
             return self._client.set(key, unique)
 
+def status_code(request,str):
```

Comment:
``status_code`` should be rename ``statsd_count``

``str`` is a reserved keyword in python, please use ``count_key`` instead.

You need a space betwee ``,`` and ` str``